lib/internal/errors.js

```diff
@@ -262,13 +270,11 @@ class AssertionError extends Error {
     if (message != null) {
       super(message);
     } else {
-      if (util === null) {
-        util = require('util');
-        if (process.stdout.isTTY && process.stdout.getColorDepth() !== 1) {
-          green = '\u001b[32m';
-          white = '\u001b[39m';
-          red = '\u001b[31m';
-        }
+      const util = lazyUtil();
+      if (process.stdout.isTTY && process.stdout.getColorDepth() !== 1) {
```

Comment:
Just pointing out that this was only executed once in the past, and is now executed each time through the code path.

Reply:
Good catch, I'll fix it